README.md

```diff
@@ -638,47 +658,50 @@ mean_review_count(restaurant_list)
 
 
 
-Next, let's maybe look at the median review, since we want to make sure that there aren't any outliers in our data. Ideally, the median and mean will be somewhat close, but obviously this would be more accurate given a larger sample size. Define a function `median_review_count()` that again takes in a list of restaurant dictionaries and returns the median count of reviews. Remember that if a dataset contains even number of data points, to get the median we average the two middle data points.
+Now we have an idea of how many reviews a typical restaurant has, but none of these restaurants have exactly that number of reviews.
+
+Which restaurants have a `review_count` within 150 of the average?
+
+Return a list of the restaurant names.
 
 
 ```python
-# code goes here
-def median_review_count(list_of_restaurants):
-    pass
+def near_average_review_count(list_of_restaurants):
+    mean = mean_review_count(list_of_restaurants)
+    # Replace None with appropriate code
+    None
 ```
 
 
 ```python
-median_review_count(restaurant_list) #1284.5
+def near_average_review_count(list_of_restaurants):
+    mean = mean_review_count(list_of_restaurants)
+    
+    results = []
+    for restaurant in list_of_restaurants:
+        review_count_difference = abs(restaurant['review_count'] - mean)
+        if review_count_difference < 150:
```

Comment:
Should be <= 150 since the question is inclusive of restaurants that are right at 150, right?

Reply:
The question language was a bit ambiguous. Clarified in the commit just added (both instructions and solution)